### DISCUSSION TITLE
Is it possible to use organicmaps in Flutter App?

### DISCUSSION THREAD

**Discussion by basit74** [organicmaps/organicmaps#3965] [Ideas]
I am developing an App which needs a possibility for Offline navigation.
I am using Flutter/Dart

Is there any possibility to use organicmaps in my app? or what will be best possible way to use Openstreet maps for Navigation in my App.

Thanks

**Comment by biodranik** [comment]
Check https://github.com/organicmaps/api-android, https://github.com/organicmaps/api-ios and https://omaps.app/api
